indexer/services/vulcan/src/config.ts

```diff
@@ -20,6 +20,13 @@ export const configSchema = {
   ...kafkaConfigSchema,
   ...redisConfigSchema,
 
+  BATCH_PROCESSING_ENABLED: parseBoolean({ default: true }),
+  KAFKA_BATCH_PROCESSING_COMMIT_FREQUENCY: parseNumber({
```

Comment:
Nit： seems not used in this change. Is this for minimum # within 1 second?

Reply:
good chance, removed!